### DISCUSSION TITLE
Notifications through Discord DMs

### DISCUSSION THREAD

**Discussion by ayykamp** [sct/overseerr#1151] [Feature Requests]
I think it would be nice to also have notifications through discord dms. Currently this isnt possible because webhooks only work in channels. So you would need a discord bot to send those notifications. Maybe it would make sense to integrate with requestrr for that?

**Comment by sct** [comment]
Requestrr already has Overseerr integration. Go check it out! 😄

**Reply by snupple** [reply]
+1 having this feature in Overseerr

**Reply by sct** [reply]
There are no plans to add this feature to Overseerr. Use one of the third-party bots.
